ingestion/src/metadata/ingestion/source/metadata/amundsen.py

```diff
@@ -495,7 +503,31 @@ def get_type_primitive_type(self, data_type):
         return data_type
 
     def get_database_service(self, service_name: str) -> DatabaseService:
-        service = self.metadata.get_by_name(entity=DatabaseService, fqn=service_name)
+        """
+        Method to get and create Database Service
+        """
+        try:
+            file_path = (
+                Path(__file__).cwd() / "examples/amundsen_data/service_type.json"
+            )
+            with open(file_path, encoding="UTF-8") as file:
+                service_type: dict = json.load(file)
+
+        except FileNotFoundError:
+            raise Exception()
```

Comment:
If we are dying due to `FileNotFoundError`, let's raise it with proper context. E.g.,

```python
except FileNotFoundError as err:
    raise FileNotFoundError("could not get service mapping file from blablabla")
```